Sources/Xcodeproj/pbxproj().swift

```diff
@@ -162,8 +162,10 @@ func xcodeProject(
     projectSettings.debug.DEBUG_INFORMATION_FORMAT = "dwarf"
     projectSettings.debug.ENABLE_NS_ASSERTIONS = "YES"
     projectSettings.debug.GCC_OPTIMIZATION_LEVEL = "0"
+    projectSettings.debug.GCC_PREPROCESSOR_DEFINITIONS = ["DEBUG=1", "$(inherited)"]
```

Comment:
This should be `projectSettings.debug.GCC_PREPROCESSOR_DEFINITIONS = ["$(inherited)", "DEBUG=1"]`

Reply:
I KNOW. but... it's the other way around in the newly created Xcode project.

Reply:
🤷‍♂️okay then